docs/components/bottom_navigation.md

```diff
@@ -0,0 +1,293 @@
+<!--docs:
+title: "Bottom navigation"
+layout: detail
+section: components
+excerpt: "Bottom navation bars allow movement between primary distinations in an app"
+iconId:
+path: /components/bottom-navigation/
+-->
+
+
+# Bottom navigation
+
+[Bottom navigation](https://material.io/components/bottom-navigation/#) bars allow movement between primary destinations in an app.
+
+![Example bottom navigation bar with four icons along the bottom: favorites, music, places, and news. The music icon is selected](assets/bottom_navigation/bottom-nav-generic.png)
+
+
+## Using bottom navigation
+
+Before you can use Material `BottomNavigationBar`s, you need to import the Material Components 
+package for Flutter: 
+```dart
+import 'package:flutter/material.dart';
+```
+
+You need to be using a [`MaterialApp`](https://api.flutter.dev/flutter/material/MaterialApp-class.html).
+
+For more information on getting started with the Material for Flutter, go to the Flutter [Material library](https://api.flutter.dev/flutter/material/material-library.html) page.
+
+## Bottom navigation example
+
+Source code API:
+
+* `BottomNavigationBar`
+    * [Class description](https://api.flutter.dev/flutter/material/BottomNavigationBar-class.html)
+    * [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/bottom_navigation_bar.dart)
+    * [Demo site](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/bottom_navigation/regular) 
+
+The following example shows a bottom navigation bar with four icons:
+
+* favorites
+* music note
+* places
+* news
+
+<img src="assets/bottom_navigation/flutter-bottom-nav-regular.png" alt="bottom navigation bar with favorites, music note, places, and news icons">
+
+```dart
+BottomNavigationBar(
+  type: BottomNavigationBarType.fixed,
+  backgroundColor: Color(0xFF6200EE),
+  selectedItemColor: Colors.white,
+  unselectedItemColor: Colors.white.withOpacity(.60),
+  selectedFontSize: 14,
+  unselectedFontSize: 14,
+  onTap: (value) {
+    // Respond to item press.
+  },
+  items: [
+    BottomNavigationBarItem(
+      title: Text('Favorites'),
+      icon: Icon(Icons.favorite),
+    ),
+    BottomNavigationBarItem(
+      title: Text('Music'),
+      icon: Icon(Icons.music_note),
+    ),
+    BottomNavigationBarItem(
+      title: Text('Places'),
+      icon: Icon(Icons.location_on),
+    ),
+    BottomNavigationBarItem(
+      title: Text('News'),
+      icon: Icon(Icons.library_books),
+    ),
+  ],
+)
+```
+
+### Anatomy and key properties
+
+The following is an anatomy diagram for the bottom navigation bar:
+
+![bottom nav anatomy diagram](assets/bottom_navigation/bottom-nav-anatomy.png)
+
+1. Container
+1. Inactive icon
+1. Inactive text label
+1. Active icon
+1. Active text label
+
+**Container attributes**
+
+| &nbsp; | Property |
+| --- | --- |
+| **Color** | `backgroundColor` |
+| **Stroke color** | N/A |
```

Comment:
I'm leaning towards removing all the N/A attributes but either is fine

Reply:
I think I'll leave them in since they were included in the template, so I assume they'll be on every platform's docs.

Reply:
This was pulled from other container attribute tables. If an attribute doesn't apply to this implementation, feel free to remove it.